src/makeRobot.js

```diff
@@ -37,8 +37,69 @@
  *
  * @return {Robot}
  */
+
 function makeRobot(name, wheels, version) {
-  // write code here
+  const newRobot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
```

Comment:
when key and value has same name we can write it like this 
```suggestion
const newRobot = {
    name,
    wheels,
    version,

    coords: {
      x: 0,
      y: 0,
    },
```